modules/deployment-model-impl/src/test/java/org/mule/runtime/module/deployment/impl/internal/application/ApplicationDescriptorFactoryTestCase.java

```diff
@@ -91,6 +91,11 @@ private static File getResourceFile(String resource) {
                                                                         .getAbsolutePath());
 
   @Rule
+  public SystemProperty remoteMavenRepository =
+      new SystemProperty("muleRuntimeConfig.maven.repositories.mulesoftPublic.url",
```

Comment:
Closing this PR for now since making the repo private and per executor in CB seems to make it work.